WordPress/src/main/java/org/wordpress/android/ui/plugins/PluginDetailActivity.java

```diff
@@ -245,11 +241,19 @@ public void onSitePluginVersionUpdated(PluginStore.OnSitePluginVersionUpdated ev
             return;
         }
         mPlugin = mPluginStore.getSitePluginByName(mSite, mPlugin.getName());
+        checkIfPluginIsNull();
         refreshViews();
     }
 
     // Helpers
 
+    private void checkIfPluginIsNull() {
+        if (mPlugin == null) {
+            ToastUtils.showToast(this, R.string.plugin_not_found, Duration.SHORT);
+            finish();
```

Comment:
This won't work because `finish()` doesn't immediately abort - any code in the method that called this will continue to execute.

Reply:
Fixed in 5c14ea0a1419f3253bddfc286878aa068c945f28